modules/virt-initiating-vm-migration-web.adoc

```diff
@@ -22,5 +22,11 @@ The *Migrate* action is visible to all users but only cluster administrators can
 .Procedure
 
 . Navigate to *Virtualization* -> *VirtualMachines* in the web console.
-. Select *Migrate* from the Options menu {kebab} beside a VM.
-. Click *Migrate*.
+. Take either of the following steps:
+* Click the Options menu {kebab} beside the VM you want to migrate, hover over the *Migrate* option, and select *Compute*.
++
+* Open the *VM details* page of the VM you want to migrate, click the *Actions* menu, hover over the *Migrate* option, and select *Compute*.
+. In the *Migrate Virtual Machine to a different Node* dialog box, select either the *Automatically Selected Node* or the *Specific Node* option.
+. If you selected the *Automatically Selected Node* option, click *Migrate Virtual Machine*.
++
+If you selected the *Specific Node* option, choose the required node from the list displayed in the dialog box and click *Migrate Virtual Machine*.
```

Comment:
This feels a tad convoluted. Here's a suggestion to simplify it:

```suggestion
. In the *Migrate Virtual Machine to a different Node* dialog box, select either the *Automatically Selected Node* or the *Specific Node* option.
.. If you selected the *Specific Node* option, select a node from the list.
. Click *Migrate Virtual Machine*.
```

Reply:
@ousleyp  Thank you, I made the changes. Could you merge it?